js/src/lib/components/DropdownMenu/DropdownMenu.svelte

```diff
@@ -42,12 +48,20 @@
 			onClose();
 		}
 	}
+
+	function handleResize() {
+		if (docWidth !== document.documentElement.clientWidth) {
+			onClose();
+		}
+	}
 </script>
 
+<svelte:window on:resize={handleResize} on:blur={onClose} />
```

Comment:
what happens on scroll ? or is the screen unscrollable while the dropdown is open ?

Reply:
Nothing, the menu stays in place unless you click somewhere else in the document, you resize or leave the window
![May-16-2023 15-07-49](https://github.com/huggingface/hub-docs/assets/468620/d7f2f46c-3273-443c-a170-9a745b1d7ff9)